src/Interpreters/ProcessList.cpp

```diff
@@ -306,7 +306,7 @@ ProcessListEntry::~ProcessListEntry()
 
 
 QueryStatus::QueryStatus(
-    ContextPtr context_, const String & query_, const ClientInfo & client_info_, QueryPriorities::Handle && priority_handle_, const String & query_kind_)
+    ContextPtr context_, const String & query_, const ClientInfo & client_info_, QueryPriorities::Handle && priority_handle_, const IAST::QueryKind & query_kind_)
```

Comment:
If IAST::QueryKind is just enum we can pass it directly by value, not by reference. It is not necessary to change it.